dashboards/azure-anomaly-detector/azure-anomaly-detector.json

```diff
@@ -0,0 +1,293 @@
+{
+  "name": "Azure Anomaly Detector",
+  "description": null,
+
+  "pages": [
+    {
+      "name": "Cognitive Service Anomaly Detector",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Total calls",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 3,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT SUM(`azure.cognitiveservices.accounts.TotalCalls`) AS 'Total Calls' WHERE collector.name='azure-monitor' AND azure.resourceType='microsoft.cognitiveservices/accounts' AND azure.cognitiveservices.accounts.apiname LIKE '%Anomaly Detector%'"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "WARNING"
+              }
+            ]
+          }
+        },
+        {
+          "title": "Total errors",
+          "layout": {
+            "column": 4,
+            "row": 1,
+            "width": 3,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT SUM(`azure.cognitiveservices.accounts.TotalErrors`) AS 'Total Errors' WHERE collector.name='azure-monitor' AND azure.resourceType='microsoft.cognitiveservices/accounts' AND azure.cognitiveservices.accounts.apiname LIKE '%Anomaly Detector%'"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "CRITICAL",
+                "value": 0
+              }
+            ]
+          }
+        },
+        {
+          "title": "",
+          "layout": {
+            "column": 7,
+            "row": 1,
+            "width": 6,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "# Azure Anomaly Detector\nAnomaly Detector is an AI service with a set of APIs, which enables you to monitor and detect anomalies in your time series data with little machine learning (ML) knowledge, either batch validation or real-time inference."
```

Comment:
Could this widget describing the dashboard be moved to the top left corner of the dashboard?

Reply:
this is the standard being followed across all azure entities for project northstar which are being merged and live. We intend to follow similar template for the rest of the services